src/test/groovy/graphql/validation/rules/ExecutableDefinitionsTest.groovy

```diff
@@ -46,7 +47,7 @@ class ExecutableDefinitionsTest extends Specification {
     }
 
     def 'Executable Definitions with type definition'() {
-        def query = """
+        def query = StringGroovyMethods.stripIndent("""
```

Comment:
Java 15 introduced a `stripIndent` method which is used by default now. In order to keep the tests the same we now specify the groovy method explicitly